docs/api/app.md

```diff
@@ -635,8 +635,18 @@ Returns `string` - The current application directory.
     * `%APPDATA%` on Windows
     * `$XDG_CONFIG_HOME` or `~/.config` on Linux
     * `~/Library/Application Support` on macOS
-  * `userData` The directory for storing your app's configuration files, which by
-    default it is the `appData` directory appended with your app's name.
+  * `userData` The directory for storing your app's configuration files, which
+    by default is the `appData` directory appended with your app's name. By
+    convention files storing user data should be written to this directory, and
+    it is not recommended to write large files here because some environments
+    may backup this directory to cloud storage.
```

Comment:
Does Chromium have this problem? I would be surprised if Chrome's caches are uploaded to cloud storage on any platform...

Reply:
Hang on—so Chrome can choose the session data directory per-profile, then! Chromium will put things under `userData/Profile 1`, `userData/Profile 2` and so on. Shouldn't we offer the same capability?

Reply:
Hm, Chrome writes to `User Data/$partitionName`, right? `Default` is a name of a partition.

Reply:
ahhhh, right, got it. Thanks, that makes sense!